library/cohesity_oracle.py

```diff
@@ -0,0 +1,415 @@
+#!/usr/bin/python
+# Copyright (c) 2018 Cohesity Inc
+# Apache License Version 2.0
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+import time
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.urls import open_url, urllib_error
+
+try:
+    # => When unit testing, we need to look in the correct location however, when run via ansible,
+    # => the expectation is that the modules will live under ansible.
+    from module_utils.storage.cohesity.cohesity_auth import get__cohesity_auth__token
+    from module_utils.storage.cohesity.cohesity_utilities import cohesity_common_argument_spec, raise__cohesity_exception__handler, REQUEST_TIMEOUT
+    from module_utils.storage.cohesity.cohesity_hints import get__prot_source__all
+except Exception as e:
+    from ansible.module_utils.storage.cohesity.cohesity_auth import get__cohesity_auth__token
+    from ansible.module_utils.storage.cohesity.cohesity_utilities import cohesity_common_argument_spec, raise__cohesity_exception__handler, REQUEST_TIMEOUT
+    from ansible.module_utils.storage.cohesity.cohesity_hints import get__prot_source__all
+
+DOCUMENTATION = '''
+module: cohesity_oracle
+short_description: Management of Cohesity Protection Sources
+description:
```

Comment:
can remove this documentation from this script, as we will document it separately

Reply:
Okay